README.md

```
@@ -1,19 +1,324 @@
-`npm install`
+## **Steps to Complete the Project:**
 
-`node app.js`
+### **1\. Login to AWS and Create an EC2 Instance**
 
-or Run by docker compose
+* **Login to your AWS account** through the [AWS Console](https://aws.amazon.com/console/).
+    
+* **Launch an EC2 instance**:
+    
+    * Navigate to the **EC2 Dashboard** and click on **Launch Instance**.
```

Markdownlint

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)